stdlib/ballerina-builtin/src/main/ballerina/ballerina/util/natives.bal

```diff
@@ -31,3 +31,13 @@ public native function base16ToBase64Encode (string baseString) (string);
 @Param {value:"s: string to be encoded"}
 @Return {value:"the encoded string."}
 public native function base64ToBase16Encode (string baseString) (string);
+
+@Description {value:"Encodes a base10 encoded string to url and filename safe base64."}
+@Param {value:"s: string to be encoded"}
+@Return {value:"the encoded string."}
```

Comment:
Shall we capitalize the first letter of the description? i.e The encoded string.